app/routes.test.js

```diff
@@ -397,4 +402,23 @@ describe('routes', () => {
           expect(res.text.includes('<h1 class="nhsuk-heading-l nhsuk-u-padding-left-3" data-test-id="error-page-title">Error: Incorrect url /aaaa - please check it is valid and try again</h1>')).toEqual(true);
         })));
   });
+
+  // TODO: Remove test below when capabilities selector is on by default
+  describe('when capabilities selector is off by default', () => {
+    beforeEach(() => {
+      config.useCapabilitiesSelector = false;
+    });
+    it('should return the correct status and text if there is no error for capabilities-selector ', () => {
```

Comment:
change to -> return the correct status and redirect to `capabilities-selector.all`  if there is no errors